### ISSUE TITLE
[Image] Transparent Images in Dark Mode

### ISSUE THREAD

**Issue by xinemata** [processing/p5.js-website#887]
### Title

[Image] Transparent Images in Dark Mode

### Description

In dark mode, images with transparent backgrounds may not be visible. 

### Steps to Reproduce

1. Go to https://p5js.org/contribute/fes_contribution_guide/
2. Locate the Accessibility drop-down menu, select Dark Mode
3. <img width="1196" alt="Image" src="https://github.com/user-attachments/assets/5233cb78-11fe-4dff-b972-cd49fc28d5ef" />

### Actual Behavior

The transparent image is barely visible in dark mode.

### Expected Behavior

Image content and background color should have a minimum of 4.5:1 contrast ratio. 

### Environments

Browser & Version: Chrome 138
Operating System & Version: macOS 15.1 

### Suggested Fix

- Possibly update CSS to apply invert() to dark mode images
- Or ensure that images on the website don't have a transparent background

### Reference

[1.4.3 Contrast (Minimum) Level AA](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum)

### What is your operating system?

Mac OS

### Web browser and version

Chrome 138

**Comment by reshma045** [comment]
@xinemata I'd like to work on this issue. Can you assign it to me?

**Comment by ugec67** [comment]
Hello maintainers! 👋 I’m a first-time contributor and very interested in contributing to this issue. Please let me know if I can take this up, and kindly assign it to me if it’s still available. Thank you in advance!

**Comment by perminder-17** [comment]
Hi @reshma045 and @ugec67, thank you both for showing interest in this issue. Since @reshma045 expressed interest first, I’ll go ahead and assign it to you. @ugec67, once a PR is created, it would be great if you could help by reviewing it. Thanks again both!

**Comment by reshma045** [comment]
Thanks @perminder-17

**Comment by reshma045** [comment]
@perminder-17 I've linked a PR that solves this issue. Thanks
